android/src/main/AndroidManifest.xml

```diff
@@ -7,5 +7,6 @@
             <action android:name="android.intent.action.VIEW" />
         </intent>
     </queries>
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```

Comment:
Adding this permission to the Manifest is not enough. We also need to show the permission request dialog to the user and handle the user's response.
It's already tracked in #710, so let's remove this line.

Reply:
With the downgrade to 32 this is not required so sure...